bin/install-qa-check.d/60pkgconfig

```diff
@@ -75,6 +75,29 @@ pkgconfig_check() {
 		eqatag -v pkgconfig.bad-libdir "${bad_libdir[@]}"
 	fi
 
+	# Check for mismatched Version field vs ${PV}
+	# To be safe, let's make sure _all_ installed .pcs have a bad Version
+	# before warning, as this should catch the general cases we're worried
+	# about, while avoiding any pathological cases e.g. multiple libraries
+	# with different versioning within one package.
+	# Example bugs: bug #833895, bug #833887.
+	local all_bad=yes
+	# Record the last bad file matched
+	local bad_file
+	for f in "${files[@]}" ; do
+		if [[ ${PV} == $(sed -n -e 's/^Version: //p' "${f}" || die) ]] ; then
```

Comment:
There's a problem here where ${PV} has e.g. _p1 for say, a Debian patchset, and it obviously then won't match the .pc file. Not sure how we should handle that best.

Reply:
Meh. For now, I might just skip it if the version contains _p* (including _pre) given that I don't think we necessarily expect correct versions there anyway and it complicates the logic. QA checks are intended to be overall wins rather than perfect, and so avoiding the false positives is more important than trying to catch every case IMO.